doc/sphinx/particles.rst

```diff
@@ -654,7 +654,7 @@ Lattice-Boltzmann swimmers
 
     import espressomd
     system = espressomd.System(box_l=[1, 1, 1])
-    system.part.add(pos=[2, 0, 0], rotation=[1, 1, 1], swimming={
+    system.part.add(pos=[2, 0, 0], rotation=[True, True, True], swimming={
```

Comment:
``3 * [True]`` ?

Reply:
I once tried to change these and box_l to the `3 * [x]` syntax, but I got push back from the core team. Some people prefer the `[x] * 3` syntax, others prefer `[x, x, x]`. I guess it's a matter of taste.